protocol/messenger_contacts.go

```diff
@@ -153,27 +153,45 @@ func (m *Messenger) AcceptContactRequest(ctx context.Context, request *requests.
 		return nil, err
 	}
 
-	err = m.syncContactRequestDecision(ctx, request.ID.String(), true, m.dispatchMessage)
+	err = m.syncContactRequestDecision(ctx, request.ID.String(), "", true, m.dispatchMessage)
 	if err != nil {
 		return nil, err
 	}
 
 	return response, nil
 }
 
-func (m *Messenger) declineContactRequest(requestID string, fromSyncing bool) (*MessengerResponse, error) {
+func (m *Messenger) declineContactRequest(requestID, contactID string, fromSyncing bool) (*MessengerResponse, error) {
 	m.logger.Info("declineContactRequest")
-	contactRequest, err := m.persistence.MessageByID(requestID)
-	if err != nil {
-		return nil, err
-	}
 
-	contact, err := m.BuildContact(&requests.BuildContact{PublicKey: contactRequest.From})
+	var (
```

Comment:
Is there a reason why you are using the `var` block instead of just `contactRequest, err := m.persistence.MessageByID(requestID)` (with the colon)? Same question for the function `updateAcceptedContactRequest`.